sidebars.js

```diff
@@ -851,6 +852,841 @@ const sidebars = {
+                {
+                  type: 'doc',
+                  id: 'run-transactions-through-scalardb-core-library',
+                  label: 'ScalarDB Core ライブラリーを使用',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalardb-cluster/run-transactions-through-scalardb-cluster',
+                  label: 'ScalarDB Cluster を使用',
+                },
+              ],
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/run-transactions-through-scalardb-cluster-sql',
+              label: 'SQL インターフェースを使用して実行する',
+            },
+            {
+              type: 'category',
+              label: '高度な設定と操作',
+              collapsible: true,
+              items: [
+                {
+                  type: 'doc',
+                  id: 'scalardb-cluster/scalardb-auth-with-sql',
+                  label: 'SQL を使用してユーザーを認証および承認する',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalardb-benchmarks/README',
+                  label: 'ベンチマークを実行する',
+                },
+              ],
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: '分析クエリを実行する',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'scalardb-samples/scalardb-analytics-postgresql-sample/README',
+              label: 'ScalarDB Analytics with PostgreSQL を実行する',
+            },
+            // This doc no longer exists but we should have some type of doc here.
+            // {
+            //   type: 'doc',
+            //   id: 'scalardb-analytics-spark/getting-started',
+            //   label: 'Run ScalarDB Analytics',
+            // },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'Run Non-Transactional Storage Operations',
+          collapsible: true,
+          items: [
+            {
+              type: 'category',
+              label: '非トランザクションストレージ操作を実行する',
+              collapsible: true,
+              items: [
+                {
+                  type: 'doc',
+                  id: 'run-non-transactional-storage-operations-through-library',
+                  label: 'ScalarDB Core ライブラリーを使用する',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalardb-cluster/run-non-transactional-storage-operations-through-scalardb-cluster',
+                  label: 'ScalarDB Cluster を使用する',
+                },
+              ],
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/run-non-transactional-storage-operations-through-sql-interface',
+              label: 'SQL インターフェースを使用して実行する',
+            },
+            {
+              type: 'doc',
+              id: 'run-non-transactional-storage-operations-through-primitive-crud-interface',
+              label: 'プリミティブ CRUD インターフェースを使用して実行する',
```

Comment:
Directly translated `プリミティブ` sounds a bit odd to me. Maybe we don't need to mention `primitive` in this case?
```suggestion
              label: 'CRUD インターフェースを使用して実行する',
```